applications/DamApplication/custom_processes/dam_bofang_condition_temperature_process.hpp

```diff
@@ -11,8 +11,8 @@
 //
 //
 
-#if !defined(KRATOS_BOFANG_CONDITION_TEMPERATURE_PROCESS )
-#define  KRATOS_BOFANG_CONDITION_TEMPERATURE_PROCESS
+#if !defined(KRATOS_DAM_BOFANG_CONDITION_TEMPERATURE_PROCESS )
+#define  KRATOS__DAM_BOFANG_CONDITION_TEMPERATURE_PROCESS
```

Comment:
Shouldn't KRATOS__DAM_BOFANG_CONDITION_TEMPERATURE_PROCESS be KRATOS_DAM_BOFANG_CONDITION_TEMPERATURE_PROCESS ?